.vscode/launch.json

```diff
@@ -303,6 +303,26 @@
             "request": "launch",
             "skipFiles": ["<node_internals>/**"],
             "type": "pwa-node"
+        },
+        {
+            "args": [
+                "--require",
+                "ts-node/register",
+                "test/**/*.ts",
+                "--timeout",
+                "10000",
+                "--colors",
+                "--extension",
+                "ts",
+                "${workspaceFolder}/packages/binding-firestore/test"
+            ],
+            "internalConsoleOptions": "openOnSessionStart",
+            "cwd": "${workspaceFolder}/packages/binding-firestore/",
+            "name": "Firestore Tests",
+            "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
+            "request": "launch",
+            "skipFiles": ["<node_internals>/**"],
+            "type": "pwa-node"
```

Comment:
Question: shall we add this extension?
@relu91 ?

Reply:
it is fine. Since also the other pages has their dedicated launch task for tests it make sense to have one also for firestore